apps/explorer/src/ui/CoinBalance.tsx

```diff
@@ -1,21 +1,38 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { Heading } from '~/ui/Heading';
-import { Text } from '~/ui/Text';
+import {
+    useFormatCoin,
+    CoinFormat,
+    formatBalance,
+} from '~/hooks/useFormatCoin';
+import { Amount } from '~/ui/Amount';
 
-// Passing amount as a string or number for optional number suffix
-export type CoinBalanceProps = { amount: number | string; symbol?: string };
+export interface CoinBalanceProps {
```

Comment:
This should really just extend amount props.

Reply:
Done